tools/serverpod_cli/lib/src/commands/upgrade.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io';
+
+import 'package:serverpod_cli/src/logger/logger.dart';
+import 'package:serverpod_cli/src/runner/serverpod_command.dart';
+
+import '../generated/version.dart';
+
+class UpgradeCommand extends ServerpodCommand {
+  @override
+  final name = 'upgrade';
+
+  @override
+  final description = 'Upgrade Serverpod to the latest version.';
+
+  @override
+  void run() async {
+    dynamic error;
+    var success = await log.progress(
+        'Running `dart pub global activate serverpod_cli`...', () async {
```

Comment:
Suggestion: Rephrase to "Updating Serverpod Cli...".

This message should be in a `log.debug` inside the progress call as it more explains the steps taken.

Reply:
For some reason `log.debug` does not present any information to the user.
<img width="1168" alt="image" src="https://github.com/serverpod/serverpod/assets/34163765/75476ddb-86b4-4eda-a517-b489b3aa9458">

However, if I use log.info
<img width="1170" alt="image" src="https://github.com/serverpod/serverpod/assets/34163765/fb5e5638-b6d3-435c-b496-02f655f82942">

Reply:
Sorry, I think my suggestion was unclear.

The `log.progress` should remain with a rephrased message saying "Updating Serverpod Cli...". Then the old message "Running `dart pub global activate serverpod_cli`" can be placed in a log debug inside the `log.progress`. That makes it possible for a user to see the details of what is happening if they run the CLI with the `--verbose` or `-v` flag, as that is the only case when `log.debug` prints are shown.

Reply:
It makes sense now, thank you for your patience 😃